app/ResolveRoute.js

```diff
@@ -7,8 +7,9 @@ export const routeRegex = {
     CategoryFilters: /^\/(hot|created|trending|active|promoted)\/?$/ig,
     PostNoCategory: /^\/(@[\w\.\d-]+)\/([\w\d-]+)/,
     Post: /^\/([\w\d\-\/]+)\/(\@[\w\d\.-]+)\/([\w\d-]+)\/?($|\?)/,
+    PostJson: /^\/([\w\d\-\/]+)\/(\@[\w\d\.-]+)\/([\w\d-]+)(\.json)$/,
     UserJson: /^\/(@[\w\.jso\d-]+)(\.json)$/,
```

Comment:
@mcifani this is something I missed in your other PR -- what's up with `jso` in the regex on this line? It looks out of place